tests/test_checks_googlefonts.py

```diff
@@ -3553,22 +3553,22 @@ def test_check_STAT_gf_axisregistry():
 
     # Let's add a MORF Axis with custom axisvalues
     stat = [
-        dict(
-            tag="MORF",
-            name="Morph",
-            values=[
-                dict(name="Foo", value=0),
-                dict(name="Bar", value=100),
+        {
+            "tag": "MORF",
+            "name": "Morph",
+            "values": [
+                {"name": "Foo", "value": 0},
+                {"name": "Bar", "value": 100},
             ],
-        ),
-        dict(
-            tag="wght",
-            name="Weight",
-            values=[
-                dict(name="Regular", value=400, flags=0x2),
-                dict(name="Bold", value=700),
+        },
+        {
+            "tag": "wght",
+            "name": "Weight",
+            "values": [
+                {"name": "Regular", "value": 400, "flags": 0x2},
+                {"name": "Bold", "value": 700},
             ],
-        ),
+        },
```

Comment:
;-)

Reply:
Thanks, @m4rc1e

I'm about to cut a new pre-release and I want to include this. Please just add one more commit with a changelog entry and then I'll take care of all the rest. Don't even bother about the ongoing build getting green, I'll take care of that as well.

Reply:
Sure thing

Reply:
ok, great! I'm taking over to merge this now.